homeassistant/components/tasmota/discovery.py

```diff
@@ -68,7 +68,9 @@ async def _discover_entity(tasmota_entity_config, discovery_hash, platform):
                 tasmota_entity_config,
             )
         else:
-            tasmota_entity = tasmota_get_entity(tasmota_entity_config, tasmota_mqtt)
+            tasmota_entity = tasmota_get_entity(
+                tasmota_entity_config, tasmota_mqtt, hass.async_create_task
```

Comment:
It's weird that we share our API with Tasmota library. The library can schedule its tasks with asyncio. Isn't that enough?

Reply:
Yes it's enough, fixed.